### PR TITLE
Adds a view controller property to the RNTBrightcove view component, …

### PR DESCRIPTION
**Pull request by mabidakun**
Adds a view controller property to the RNTBrightcove view component, which it can then use to perform fullscreen, auto-rotating presentation of the video player.

### CODE REVIEW

File: packages/brightcove-video/ios/RNTBrightcove/RNTBrightcove.m [hunk 1]
```diff
@@ -4,12 +4,14 @@
 #import <React/RCTLog.h>
 
 #import "RNTBrightcove.h"
+#import "storybooknative-Swift.h"
```
Comment (RussellToon) [inline packages/brightcove-video/ios/RNTBrightcove/RNTBrightcove.m]: Is this necessary / relevant?

Comment (dickie81) [reply packages/brightcove-video/ios/RNTBrightcove/RNTBrightcove.m]: yes - but won't work in another project's context

Comment (mabidakun) [reply packages/brightcove-video/ios/RNTBrightcove/RNTBrightcove.m]: Good spot, @RussellToon.

I've now submitted an update, which migrates the view controller from Swift to Objective-C - taking advantage of the absence of a this sort of modularisation (!) in Objective-c. 🤓

**Comment by mabidakun** [comment]
@dickie81 & @RussellToon  - Please review again.

File: ios/storybooknative/RNTFullscreenPresentingAutoRotatingViewController.m [hunk 2]
```diff
@@ -29,7 +29,7 @@ - (BOOL)shouldAutorotate {
 
 - (UIInterfaceOrientationMask)supportedInterfaceOrientations {
 
-  return UIInterfaceOrientationMaskAll;
+  return UIInterfaceOrientationMaskAllButUpsideDown;
```
Comment (trevorah) [inline ios/storybooknative/RNTFullscreenPresentingAutoRotatingViewController.m]: 🙃

Comment (mabidakun) [reply ios/storybooknative/RNTFullscreenPresentingAutoRotatingViewController.m]: 🤓

**Pull request merged**